expat/configure.ac

```diff
@@ -60,7 +60,7 @@ m4_define([expat_version],
   m4_ifdef([__gnu__],
            [esyscmd(conftools/get-version.sh lib/expat.h)],
            [2.2.x]))
-AC_INIT([expat], expat_version, [expat-bugs@libexpat.org])
+AC_INIT([expat],[expat_version],[expat-bugs@libexpat.org])
```

Comment:
Please add single spaces after commas to be in line with the examples in the style guide at e.g. https://github.com/libexpat/libexpat/blob/master/expat/configure-ac-style.md#2-parameter-indentation . Thanks.

Reply:
PS: This change in quoting turned out to be mistaken, would have made use produce tarballs like `expat-expat_version.tar.gz` (verbatim). So `autoupdate` is wrong about it.